app/routers/questions.py

```diff
@@ -1,29 +1,16 @@
 from fastapi import APIRouter, status, HTTPException
-from fastapi.responses import JSONResponse
-from fastapi.encoders import jsonable_encoder
 from database import client
-from models import Question
+from models import QuestionResponse
 
-router = APIRouter(prefix="/questions", tags=["questions"])
+router = APIRouter(prefix="/questions", tags=["Questions"])
 
 
-@router.get("/")
-async def get_questions():
-    questions = client.quiz.questions.find({})
-    return questions
-
-
-@router.post("/", response_model=Question)
-async def create_question(question: Question):
-    question = jsonable_encoder(question)
-    new_question = client.quiz.questions.insert_one(question)
-    created_question = client.quiz.questions.find_one({"_id": new_question.inserted_id})
-    return JSONResponse(status_code=status.HTTP_201_CREATED, content=created_question)
-
-
-@router.get("/{question_id}")
+@router.get("/{question_id}", response_model=QuestionResponse)
```

Comment:
because of the response model, when we make a get request to get a question with some `question_id`, it will not return the question ka details but just the question_set_id right?
Why is that? I just want a question so I should be able to get it?

Will we have to make another call to get question from the question_set_id?